python/chigger/tests/line_sample/line_sample_error.py

```diff
@@ -18,3 +18,4 @@
 sample.update()
 x = sample[0].getDistance()
 y = sample[0].getSample('invalid_name') # testing this error
+exit(1)
```

Comment:
@aeslaughter This test was a `RunException` test that requires a non-zero exit status.
It is either this change or change the test spec into a `RunApp` test with `expect_out` and setting `errors` to a dummy value (since there is an `ERROR` in the output). Thoughts?

Reply:
Just looking for the error in the output should be fine.